py3status/py3.py

```diff
@@ -608,28 +609,39 @@ def format_contains(self, format_string, name):
         will fail if the format string contains placeholder formatting
         eg ``'{placeholder:.2f}'``
         """
+        if isinstance(names, basestring):
+            names = [names]
 
         # We cache things to prevent parsing the format_string more than needed
-        try:
-            return self._format_placeholders_cache[format_string][name]
-        except KeyError:
-            pass
+        boolean_list = []
+        for name in names:
+            try:
+                boolean_list.append(
+                    self._format_placeholders_cache[format_string][name])
+            except KeyError:
+                pass
+        if len(names) == len(boolean_list):
+            return any(boolean_list)
```

Comment:
the old caching is much better as we don't want to do all this work

just make a key use `names` if str or `str(names)` if a list